### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by IAvecilla**
## Motivation

Wrote new documentation for the aleo sdk to use on the npm package page. We removed JsDoc support for the wasm generated code and all the configuration related to it.

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -5,13 +5,10 @@
   "main": "src/core.js",
   "license": "GPL-3.0",
   "scripts": {
-    "preinstall": "cd ../wasm && wasm-pack build --out-dir ../js/aleo-wasm --target web --scope entropy-1729",
-    "generate_docs": "jsdoc --configure jsdoc.json --verbose",
+    "preinstall": "cd ../wasm && wasm-pack build --out-dir ../js/aleo-wasm --target nodejs",
```
Comment (ilitteri) [inline js/package.json]: The `cd` could be removed

```suggestion
    "preinstall": "wasm-pack build wasm/ --out-dir ../js/aleo-wasm --target nodejs",
```

File: wasm/README.md [hunk 2]
```diff
@@ -1,8 +1,250 @@
...
+##### Verify signatures
+
+You can use the signature to also verify if a specific address of an account
+signed a certain message
+
+```js
+let private_key = new PrivateKey();
+let signature = Signature.sign(private_key, "random message");
+let address = private_key.to_address();
+signature.verify(priaddress, "random message"); // true
```
Comment (ilitteri) [inline wasm/README.md]: Is the name `priaddress` ok?

**Review by ilitteri — COMMENTED** [review]
Just added two comments.

File: js/package.json (hunk 1 above)
Comment (IAvecilla) [reply js/package.json]: That's from the original aleo-sdk code from Aleo. This directory it's not going to be used at least for now. The suggestion is valid anyways so I committed the change.

File: wasm/README.md (hunk 2 above)
Comment (IAvecilla) [reply wasm/README.md]: No it was not. Fixed! 👍

**Review by ilitteri — APPROVED** [review]
LGTM

**Review by mationorato — APPROVED** [review]
LGTM

**Pull request merged**